data/seeds/001_projects.js

```diff
@@ -0,0 +1,14 @@
+
+exports.seed = function (knex, Promise) {
```

Comment:
Excellent job using seeds.  I like that you stretched yourself to use them because you'll need to be familiar with them anyways.